nbodykit/algorithms/pair_counters/simbox.py

```diff
@@ -0,0 +1,199 @@
+    results.
+
+    .. note::
+
+        The algorithm expects the positions of particles in a simulation box to
+        be the Cartesian ``x``, ``y``, and ``z`` vectors. To compute
+        pair counts on survey data, using right ascension, declination, and
+        redshift, see :class:`~nbodykit.algorithms.SurveyDataPairCount`.
+
+    Parameters
+    ----------
+    mode : '1d', '2d', 'projected', 'angular'
+        compute pair counts as a function of the specified coordinate basis;
+        see the Notes section below for specifics
+    first : CatalogSource
+        the first source of particles, providing the 'Position' column
+    edges : array_like
+        the separation bin edges along the first coordinate dimension;
+        depending on ``mode``, the options are :math:`r`, :math:`r_p`, or
+        :math:`\theta`. Expected units for distances are :math:`\mathrm{Mpc}/h`
+        and degrees for angles. Length of nbins+1
+    BoxSize : float, 3-vector, optional
+        the size of the box; if 'BoxSize' is not provided in the source
+        'attrs', it must be provided here
+    periodic : bool, optional
+        whether to use periodic boundary conditions
+    second : CatalogSource, optional
+        the second source of particles to cross-correlate
+    los : {'x', 'y', 'z'}, int, optional
+        the axis of the simulation box to treat as the line-of-sight direction;
+        this can be provided as string identifying one of 'x', 'y', 'z' or
+        the equivalent integer number of the axis
+    Nmu : int, optional
+        the number of :math:`\mu` bins, ranging from 0 to 1; requred if
+        ``mode='2d'``
+    pimax : float, optional
+        The maximum separation along the line-of-sight when ``mode='projected'``.
+        Distances along the :math:`\pi` direction are binned with unit
+        depth. For instance, if ``pimax=40``, then 40 bins will be created
+        along the :math:`\pi` direction.
+    weight : str, optional
+        the name of the column in the source specifying the particle weights
+    show_progress : bool, optional
+        if ``True``, perform the pair counting calculation in 10 iterations,
+        logging the progress after each iteration; this is useful for
+        understanding the scaling of the code
+    **config : key/value pairs
+        additional keywords to pass to the :mod:`Corrfunc` function
+
+    Notes
+    -----
+    This class can compute pair counts using several different coordinate
+    choices, based on the value of the input argument ``mode``. The choices
+    are:
+
+    * ``mode='1d'`` : compute pairs as a function of the 3D separation :math:`r`
+    * ``mode='2d'`` : compute pairs as a function of the 3D separation :math:`r`
+      and the cosine of the angle to the line-of-sight, :math:`\mu`
+    * ``mode='projected'`` : compute pairs as a function of distance perpendicular
+      and parallel to the line-of-sight, :math:`r_p` and :math:`\pi`
```

Comment:
Is pi a variable here?

Reply:
No it isn't. r_\pi?

Reply:
yes its r_\pi...seems like corrfunc and halotools refer to it as \pi